### ISSUE TITLE
hosting

### ISSUE THREAD

**Issue by lanbau** [Aman-Preet-Singh-Gulati/background-removal-image#1]
hi aman, is it possible to host this on a flask api?

**Comment by Aman-Preet-Singh-Gulati** [comment]
Yes it is definitely possible I'm working on it and soon will release that version of the same with tutorial ( article )

Thanks.
